APMrover2/mode_rtl.cpp

```diff
@@ -35,8 +35,7 @@ void ModeRTL::update()
         calc_throttle(calc_reduced_speed_for_turn_or_distance(_desired_speed), true);
     } else {
         // we've reached destination so stop
-        g2.motors.set_throttle(g.throttle_min.get());
-        g2.motors.set_steering(0.0f);
+        stop_vehicle();
         lateral_acceleration = 0.0f;
```

Comment:
sorry to keep pushing back on changes.. but if we don't set lateral_acceleration to zero then that variable will have an old out-of-date number in it because "calc_lateral_acceleration" isn't being called.  When stopping we are currently not trying to steer the value.. so I think that lateral_acceleration needs to be set to zero by something.